soroban-token-spec/src/lib.rs

```diff
@@ -106,3 +107,51 @@ impl Token {
         panic!("calling into interface");
     }
 }
+
+/// Obtains the full spec for the Token contract
+pub const fn spec_xdr() -> [u8; 2037] {
```

Comment:
We missed this, the `spec_xdr` functions are all hidden in the docs. Adding it and merging:
```suggestion
/// Returns the XDR spec for the Token contract.
#[doc(hidden)]
pub const fn spec_xdr() -> [u8; 2037] {
```